jexl-eval/src/lib.rs

```diff
@@ -164,23 +164,30 @@ impl<'a> Evaluator<'a> {
                 Ok(Value::Object(map))
             }
 
-            Expression::IdentifierSequence(exprs) => {
-                assert!(!exprs.is_empty());
-                let mut rv: Option<&Value> = Some(context);
-                for expr in exprs.into_iter() {
-                    let key = self.eval_ast(*expr, context)?;
-                    if let Some(value) = rv {
-                        rv = match key {
-                            Value::String(s) => value.get(&s),
-                            Value::Number(f) => value.get(f.as_f64().unwrap().floor() as usize),
-                            _ => return Err(EvaluationError::InvalidIndexType),
-                        };
-                    } else {
-                        break;
+            Expression::Identifier(inner) => {
+                // TODO: Make this error out if the identifier does not exist in the
```

Comment:
should we file an issue for this one?

Reply:
Aaah yeah. I think there a few cleanups we might need (including this one), especially on the error handling. Opened #8 for that.